docs/kratos/organizations/organizations.mdx

```diff
@@ -389,3 +389,157 @@ local claims = std.extVar('claims');
+protocols such as SAML.
+
+Onboarding portal aims to:
+
+- Simplify complex integrations: By providing a self-service or guided onboarding experience, they abstract away the underlying
+  technical complexities of SAML. This means IT administrators on the client side don't need to be SAML experts to connect their
+  organization's identity system.
+- Reduce onboarding time: Instead of lengthy back-and-forth communication and manual configuration, an onboarding portal link
+  allows an external administrator to configure the connection directly through a user-friendly interface. This drastically
+  reduces the time and effort required to establish new B2B integrations.
+- Enhance security and control: Onboarding portal links are designed to be shareable without exposing sensitive information. The
+  administrator receiving the link can configure their side of the connection securely, maintaining control over their IdP
+  settings while establishing a trusted connection with your application.
+- Enable Self-Service for Organizations: For SaaS providers, this means empowering your customers to self-onboard and manage their
+  own SSO connections, freeing up your engineering and support teams. Support Scalability for B2B Models: As your application
+  grows and serves more enterprise clients, manually managing each integration becomes unsustainable. These features provide a
+  scalable solution for onboarding and managing a large number of organizational customers.
+- Validate SSO Connections instantly: The onboarding portal includes the ability to immediately test the configured SSO
+  connection. This allows IT administrators to verify that the integration works as expected before deploying it to their users,
+  reducing troubleshooting time and ensuring a smooth rollout.
+
+These features are relevant for:
+
+- SaaS (Software-as-a-Service) providers: SaaS companies offer services to other businesses (B2B) or to businesses that then serve
+  their own customers (B2B2C). This is crucial for enabling enterprise clients to use their existing corporate identity systems
+  (for example, Microsoft Entra ID, Okta, Google Workspace) for seamless access to the SaaS application.
+- Enterprise IT administrators: These individuals in your customer organizations are responsible for managing their internal
+  identity providers and user directories. They use the onboarding portal to configure the connection to your application.
+- Developers and architects building multi-tenant applications: These individuals implement robust and scalable identity
+  management for applications that serve multiple organizations. Each organization has its own authentication and user
+  provisioning requirements.
+
+Current limitations:
+
+- The initial version includes a guided walkthrough for a limited set of providers. OIDC support and additional providers are
```

Comment:
I would specify it currently support SAML providers and list the ones if we know are not compliant to be more transparent.